api/envoy/extensions/pattern_template/match/v3/pattern_template_match.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package envoy.extensions.pattern_template.match.v3;
```

Comment:
Sorry, one last nit here. You've created two new extension categories, but are placing two concrete extension instances for each under a common tree. I would instead organize this as two trees:
* `envoy.extensions.path.match`
* `envoy.extensions.path.rewrite`

Then you have the two examples/default instances underneath these as:
* `envoy.extensions.path.match.pattern_template.v3.`
* `envoy.extensions.path.rewrite.pattern_template.v3`

I.e. you have two generic path match/rewrite extension categories, and the "pattern template" syntax in each case is just an instantiation of the category.

Reply:
Sounds good. Switched over

Reply:
Sort of - you should have pattern_template as a path component as suggested ^^ to make clear that this is one example of a "path match" or a "path rewrite".

Reply:
I believe this is switched. Comment is on the old file path.

Reply:
Ok, think I got it right now. Added the additional directory pattern_template